js/ui/file_manager/ui.file_manager.toolbar.js

```diff
@@ -342,8 +333,23 @@ class FileManagerToolbar extends Widget {
         this._commandManager.executeCommand(command);
     }
 
-    _isCommandAvailable(commandName, fileItems) {
-        return ALWAYS_VISIBLE_TOOLBAR_ITEMS.indexOf(commandName) > -1 || this._commandManager.isCommandAvailable(commandName, fileItems);
+    _isCommandAvailable(command, fileItems) {
```

Comment:
The signature of this method should be one of:
`_isCommandAvailable(commandName, visibilityMode, fileItems)`
or
`_isToolbarItemAvailable(toolbarItem, fileItems)`